glean-core/src/error_recording.rs

```diff
@@ -100,7 +100,7 @@ pub fn test_get_num_recorded_errors(
     meta: &CommonMetricData,
     error: ErrorType,
     ping_name: Option<&str>,
-) -> i32 {
+) -> Result<i32, String> {
```

Comment:
Normally we do have a dedicated error type, but given that this should be test only and we therefore don't _want_ to expose the error in general, opting for a String error here seems ok.